boards/kws/pico_spe/pico_spe.dts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2024 KWS Computersysteme Gmbh
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include "pico_spe-common.dtsi"
```

Comment:
not sure why there is a common file when there is only one board, put contents of common file here and remove common file